docs/source/installation.rst

```diff
@@ -51,3 +51,10 @@ cloned this repo, run the following:
 .. code-block:: bash
 
    python3 -m pip install -r requirements.txt
+
+.. warning::
+    If you want to contribute to ``manim-community`` and have clone the
+    repository to your local device, please uninstall the pip-installed version
+    of ``manim-community``, if you had installed it previously.
+    This is to avoid any accidental usage of the pip-installed version when developing
+    and testing on your local copy of the repository.
```

Comment:
Good catch. We should also add language (for both users and devs) to uninstall the 3b1b version, if they have it.

Reply:
3b1b needn't be uninstalled. Because it's namespace is `manimlib` and not `manim`.

Reply:
Oh good point, I forgot.